front-end/src/context/tokenContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, useContext, useState, useEffect} from "react"
+import {baseURL, client} from "../config/AxiosConfig";
+import Cookies from 'universal-cookie';
+
+export type userTokenContent = {
+  user_token: string
+  setUserToken:(c: string) => void
+}
+
+export const MyTokenContext = createContext<userTokenContent>({
+    user_token: '',
+    setUserToken: () => {},
+})
+
+export default function Provider({children}:any) {
```

Comment:
instead of using any as the type for the Provider component's children prop, you could use React.ReactNode to indicate that it can accept any valid React component.